testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -2166,6 +2217,147 @@ QUnit.module('Options', {
             });
         });
     });
+
+    QUnit.module('ViewsCount', {
```

Comment:
seems like not all tests in this module are related to "views count"

Reply:
renamed to `ViewsCount = 2`